community/kernel/src/test/java/org/neo4j/kernel/builtinprocs/AwaitIndexProcedureTest.java

```diff
@@ -240,162 +217,4 @@ public void shouldTimeoutIfTheIndexTakesTooLongToComeOnline()
-
-        @Override
-        public void markForTermination( Status reason )
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long lastTransactionTimestampWhenStarted()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long lastTransactionIdWhenStarted()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long startTime()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long timeout()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public void registerCloseListener( CloseListener listener )
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public Type transactionType()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long getTransactionId()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public long getCommitTime()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public Revertable overrideWith( AccessMode mode )
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-    }
-
-    private class StubStatement implements Statement
-    {
-        private final ReadOperations readOperations;
-
-        StubStatement( ReadOperations readOperations )
-        {
-            this.readOperations = readOperations;
-        }
-
-        @Override
-        public void close()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public ReadOperations readOperations()
-        {
-            return readOperations;
-        }
-
-        @Override
-        public TokenWriteOperations tokenWriteOperations()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public DataWriteOperations dataWriteOperations() throws InvalidTransactionTypeKernelException
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public SchemaWriteOperations schemaWriteOperations() throws InvalidTransactionTypeKernelException
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-
-        @Override
-        public QueryRegistryOperations queryRegistration()
-        {
-            throw new UnsupportedOperationException( "not implemented" );
-        }
-    }
 }
```

Comment:
You should add tests for escaped labels and property names, such as `:``Label with space in it``(prop)` and `:Label(``prop key with space in it``)`